### PR TITLE
COMMERCE-622 Fix upgrade process

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test

File: commerce-product-type-virtual-service/src/main/java/com/liferay/commerce/product/type/virtual/internal/upgrade/v1_1_0/CPDefinitionVirtualSettingUpgradeProcess.java [hunk 1]
```diff
@@ -48,9 +48,7 @@ protected void doUpgrade() throws Exception {
 			String template = StringUtil.read(
 				CPDefinitionVirtualSettingUpgradeProcess.class.
 					getResourceAsStream(
-						"/META-INF/com/liferay/commerce/product/type/virtual" +
-							"/internal/upgrade/v1_1_0/dependencies" +
-								"/CPDefinitionVirtualSetting.sql"));
+						"/dependencies/CPDefinitionVirtualSetting.sql"));
```
Comment (ethanbustad) [inline commerce-product-type-virtual-service/src/main/java/com/liferay/commerce/product/type/virtual/internal/upgrade/v1_1_0/CPDefinitionVirtualSettingUpgradeProcess.java]: Hey @Commando18, are you sure this String should start with a `/`? I think if you're doing a relative path to the resource it should just be `"dependencies/CPDefinitionVirtualSetting.sql"`...

Comment (alessiorendina) [reply commerce-product-type-virtual-service/src/main/java/com/liferay/commerce/product/type/virtual/internal/upgrade/v1_1_0/CPDefinitionVirtualSettingUpgradeProcess.java]: Hey @ethanbustad, you're right. I'll change it, thanks!

**Comment by alessiorendina** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Pull request merged**